packages/e2e-tests/test/e2e.spec.ts

```diff
@@ -445,9 +446,19 @@ describe('e2e', function () {
     });
 
     it('version', async function () {
-      const expected = require('../package.json').version;
+      const expectedPackageVersion: string =
+        // eslint-disable-next-line @typescript-eslint/no-var-requires
+        require('../package.json')['dependencies']['@mongosh/cli-repl'];
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+      const expectedVersionFromRepl = MONGOSH_VERSION;
+
+      expect(expectedPackageVersion).not.null;
+      expect(expectedPackageVersion).equals(expectedVersionFromRepl);
+
       await shell.executeLine('version()');
-      shell.assertContainsOutput(expected);
+      shell.assertNoErrors();
+      shell.assertContainsOutput(expectedPackageVersion);
+      shell.assertContainsOutput(expectedVersionFromRepl);
```

Comment:
Not a huge deal, but I think it would be a tad cleaner to use really compare `version()` against the output of `mongosh --version` instead of adding more explicit source dependencies on the cli-repl package contents (and even internals) here.